### DISCUSSION TITLE
how to install on linux

### DISCUSSION THREAD

**Discussion by barista1** [gpdaniels/spike-prime#37] [General]
I I need a step-by-step instruction on how to install this tool on linux - I'm a beginner.

**Comment by SpudGunMan** [comment]
What you lookin to do?

https://github.com/gpdaniels/spike-prime/blob/master/simulator/README.md

**Comment by gpdaniels** [comment]
This repository contains a few projects. If you're trying to get the simulator up and running it should be as simple as installing python and then using python to start the `simulator.py` script.

Be aware that it's very much a work in progress and doesn't do too much that is useful yet.

Similarly there's a disassembler and decompiler that can decode the mpy files found on the spike-prime/mindstorms, but this is also very unfinished.
